include/swift/SIL/MemoryLocations.h

```diff
@@ -201,10 +201,18 @@ class MemoryLocations {
   /// If true, also analyze trivial memory locations.
   bool handleTrivialLocations;
 
+  /// If true, trivial values aren't cleared or killed unless "evenTrivial=true"
```

Comment:
Actually, there is no need to make this configurable. It can be the default. Given that DestroyHoisting is not used anymore and can be deleted.

Reply:
Removed the new flag and the preexisting flag.